lib/listener/helpers.js

```diff
@@ -13,7 +13,11 @@ module.exports = function () {
   const runHelpersHook = (hook, param) => {
     if (store.dryRun) return;
     Object.keys(helpers).forEach((key) => {
-      helpers[key][hook](param);
+    	if (!helpers[key][hook]) {
+    		console.log(`${key}'s hook value is: ${helpers[key][hook]}`);
+			  return;
+		  }
+		  helpers[key][hook](param);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mixed spaces and tabs.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4932579)